PREREQS.md

```diff
@@ -0,0 +1,39 @@
+## Prerequisites
+
+### AMI Acceptance
+
+Before you start, make sure the AMI that is listed in the `backendless_chef.yaml` for your region has been accepted/approved for use in the AWS Marketplace. We are using native Amazon Linux AMIs, the most up to date list is [always here](https://aws.amazon.com/amazon-linux-ami/).
+
+### For SSL
+1. You must have a DNS Record Name and Zone Name defined prior to setup if you want SSL.
+1. You must have already created and uploaded an SSL cert to AWS. Once you've uploaded the cert, you will need the `ARN` of the cert resource. For info on how to do this, please follow the [AWS Certificate Manger](https://aws.amazon.com/certificate-manager/) docs.
+
+### Using with Automate
+
+At this time, this template does not setup Automate as part of it's deployment. However, you can configure it to point to an Automate server that you will setup afterwards, or that is already setup.
+1. You should have a token already generated, follow the [instructions here](https://docs.chef.io/data_collection.html#step-1-configure-a-data-collector-token-in-chef-automate) on how to generate an Automate token.
+1. The Chef Automate URL, for more information on how this URL is formatted, [read this](https://docs.chef.io/data_collection.html#step-2-configure-your-chef-server-to-point-to-chef-automate).
+
+### Package Versions
+
+It's recommended to have the packages downloaded and hosted locally before proceeding, an S3 bucket works well for this purpose. Once you've downloaded the correct EL7 packages and have them hosted, adjust the following variables accordingly to point to the proper URLs.
```

Comment:
could we say `S3 bucket, Artifactory/Nexus or YUM/APT repository cache` to be more inclusive?

regarding the 2nd sentence, maybe we can just replace this with some simple S3 caching instructions?   like:
```bash
mixlib-install download chef-server --platform el --platform-version 7.5 --architecture x86_64
# Starting download https://packages.chef.io/files/stable/chef-server/12.17.33/el/7/chef-server-core-12.17.33-1.el7.x86_64.rpm
# Download saved to /Users/myname/chef-server-core-12.17.33-1.el7.x86_64.rpm

aws s3 cp /Users/myname/chef-server-core-12.17.33-1.el7.x86_64.rpm s3://mybucket/package-cache/ --acl public-read
# upload: ./chef-server-core-12.17.33-1.el7.x86_64.rpm to s3://mybucket/package-cache/chef-server-core-12.17.33-1.el7.x86_64.rpm

aws s3 presign s3://mybucket/package-cache/chef-server-core-12.17.33-1.el7.x86_64.rpm | cut -d '?' -f 1
# https://mybucket.s3.amazonaws.com/package-cache/chef-server-core-12.17.33-1.el7.x86_64.rpm
# ^^ set this as your ChefServerPackage value
```